biz.aQute.repository/src/aQute/bnd/repository/maven/provider/MavenBndRepository.java

```diff
@@ -291,6 +297,12 @@ private boolean isLocal(ReleaseDTO instructions) {
 
 	private Jar getSources(Tool tool, Processor context, String path) throws Exception {
 		Jar jar = toJar(context, path);
+		if (path != null && jar == null) {
+			logger.warn(
+				"A sources path was set in the -maven-release instuction, but no sources could be found in {} (base path: {}) ",
+				path, context.getBaseURI()
```

Comment:
I think the 2nd arg should just be `options.context.getBase()`. I don't see why we would want to use a URI here. We also should not `toString()` it in case logging does not happen. Leave the `toString()` call to the logging code.